matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/api/TestMatrix.kt

```diff
@@ -44,7 +44,7 @@ import javax.inject.Inject
 /**
  * This mimics the Matrix class but using TestMatrixComponent internally instead of regular MatrixComponent.
  */
-class TestMatrix constructor(context: Context, matrixConfiguration: MatrixConfiguration) {
+internal class TestMatrix constructor(context: Context, matrixConfiguration: MatrixConfiguration) {
```

Comment:
I told you!

Reply:
CI does not like this change though. I think some other test classes have to be declared internal